backend/src/test/java/gov/cdc/usds/simplereport/service/UploadServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package gov.cdc.usds.simplereport.service;
+
+import com.fasterxml.jackson.databind.RuntimeJsonMappingException;
+import gov.cdc.usds.simplereport.db.model.Person;
+import gov.cdc.usds.simplereport.db.model.StreetAddress;
+import gov.cdc.usds.simplereport.db.repository.OrganizationRepository;
+import gov.cdc.usds.simplereport.db.repository.PersonRepository;
+import gov.cdc.usds.simplereport.db.repository.ProviderRepository;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
I feel like we had a style rule against `*` imports, though we did recently disable checkstyle on tests...

Reply:
I'll see if I can change the default in IntelliJ, I let the IDE handling my imports for me 😉